src/actions/ui.js

```diff
@@ -70,7 +71,10 @@ export function showSource(sourceId: string) {
   };
 }
 
-export function togglePaneCollapse(position: string, paneCollapsed: boolean) {
+export function togglePaneCollapse(
+  position: panelPositionType,
```

Comment:
This was a new Flow error, as `string` was not equal to the `panelPositionType` enum.